tests/test_session.py

```diff
@@ -337,7 +337,8 @@ def test_start_transcript_file_write(new_meshing_session):
     session.transcript.stop()
 
     new_stat = file_name.stat()
-    assert new_stat.st_mtime > prev_mtime or new_stat.st_size > prev_size
+    # this assertion is invalid.
```

Comment:
Streaming to the meshing session has stopped due to the switch.

@mkundu1 @prmukherj @hpohekar